pkg/kubelet/client/kubelet_client.go

```diff
@@ -93,7 +93,8 @@ func (c *KubeletClientConfig) transportConfig() *transport.Config {
 			KeyFile:  c.KeyFile,
 			KeyData:  c.KeyData,
 		},
-		BearerToken: c.BearerToken,
+		BearerToken:       c.BearerToken,
+		ConnectionTimeout: 7 * time.Second,
```

Comment:
the client built here is used by the apiserver to reach a kubelet, not the other way around